tests/trainer/test_sharded_checkpoint.py

```diff
@@ -348,3 +352,43 @@ def test_mismatch_timestamp_error(world_size, tmp_path: pathlib.Path, state_dict
             autoresume=autoresume,
             run_name=run_name,
         )
+
+
+@pytest.mark.gpu
+@world_size(2)
+@pytest.mark.parametrize('use_remote', [pytest.param(True, marks=pytest.mark.remote), False])
+@pytest.mark.parametrize('state_dict_type', ['local', 'sharded'])
+@pytest.mark.skipif(version.parse(torch.__version__) < version.parse('1.13.0'),
+                    reason='requires PyTorch 1.13 or higher')
+def test_sharded_folder(world_size,
+                        use_remote,
+                        tmp_path: pathlib.Path,
+                        state_dict_type: str,
+                        s3_bucket: str = 'mosaicml-internal-checkpoints-test/ckpt_folder_test/'):
```

Comment:
this needs to point to the official integration test bucket (which it will if you don't give it a value).